docs/en/api_watcher.md

```diff
@@ -126,6 +126,7 @@ The available methods in `OacisWatcher` class are as follows.
 
 - `#async { ... }`
     - The blocks are evaluated concurrently. In this block, you can call `await` methods.
+    - An alias method `async_do` is also available in v3.3.
```

Comment:
Is `async_do` different from `do_async`?

Reply:
Thank you. I fixed it.